src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,32 @@
+package lotto.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.List;
+import lotto.util.InputParser;
+import lotto.util.InputValidator;
+
+public class InputView {
+
+    private static final String PURCHASE_AMOUNT_PROMPT = "구입금액을 입력해 주세요.";
+    private static final String WINNING_NUMBERS_PROMPT = "당첨 번호를 입력해 주세요.";
+    private static final String BONUS_NUMBER_PROMPT = "보너스 번호를 입력해 주세요.";
+
+    public String readPurchaseAmount() {
+        System.out.println(PURCHASE_AMOUNT_PROMPT);
+        return Console.readLine();
+    }
+
+    public List<String> readWinningNumbers() {
+        System.out.println();
+        System.out.println(WINNING_NUMBERS_PROMPT);
```

Comment:
개행을 `NEW_LINE = System.lineSeparator()` 처럼 상수화 해서
```suggestion
        System.out.println(NEW_LINE + WINNING_NUMBERS_PROMPT);
```
로 나타내는 거 어떠신가요?

Reply:
앗 ! 1주차에 피드백 받았던 사항인데 이번 미션에 반영을 못했네요 ㅠ
다시 상기 시켜주셔서 감사합니다 ! 너무 좋은 의견 같아요 !